assets/dev/js/editor/components/template-library/component.js

```diff
@@ -128,15 +128,51 @@ export default class Component extends ComponentModal {
 	insertTemplate( args ) {
 		const autoImportSettings = elementor.config.document.remoteLibrary.autoImportSettings;
 
-		if ( ! autoImportSettings && args.model.get( 'hasPageSettings' ) ) {
+		// eslint-disable-next-line prefer-const
+		let { model, withPageSettings = null } = args;
```

Comment:
Use const for `model` and let for `withPageSettings` or receive them directly as parameters. But don't disable the eslint check.